src/data-workspace/data-entry-cell/entry-field-input.js

```diff
@@ -43,7 +65,75 @@ export function EntryFieldInput({ fieldname, dataElement: de, setSyncStatus }) {
         }
     }
 }
+
+InputComponent.propTypes = {
+    de: PropTypes.shape({
+        optionSet: PropTypes.shape({
+            id: PropTypes.string.isRequired,
+        }),
+        optionSetValue: PropTypes.any,
+        valueType: PropTypes.string,
+    }).isRequired,
+    sharedProps: PropTypes.object.isRequired,
+}
+
+export function EntryFieldInput({
+    fieldname,
+    dataElement: de,
+    categoryOptionCombo: coc,
+    setSyncStatus,
+}) {
+    const currentItemContext = useCurrentItemContext()
+    const rightHandPanel = useRightHandPanelContext()
+    const { id: deId } = de
+    const { id: cocId } = coc
+    const dataValueParams = useDataValueParams({ deId, cocId })
+    const dataValueSet = useDataValueSet()
+    const currentItem = useCreateCurrentItem({
+        fieldname,
+        de,
+        coc,
+        dataValueSet,
+    })
+
+    const onKeyDown = (event) => {
+        const { key, shiftKey } = event
+
+        if (shiftKey && key === 'Enter') {
+            currentItemContext.setItem(currentItem)
+            rightHandPanel.show('data-details')
+        } else if (key === 'ArrowDown' || key === 'Enter') {
+            event.preventDefault()
+            focusNext()
+        } else if (key === 'ArrowUp') {
+            event.preventDefault()
+            focusPrev()
+        }
+    }
+
+    const onFocus = () => {
+        rightHandPanel.hide()
```

Comment:
As a user, I actually find this annoying. If I want to see the sidebar-info, I most likely want to explore that information for other values/fields as well.
I'm not sure if this is in the spec, but if so I think it's something we should discuss.